sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ActivityExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using OpenTelemetry.Exporter.AzureMonitor.Models;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal static class ActivityExtensions
+    {
+        private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
+        private static readonly string INVALID_TRACE_ID = default(ActivityTraceId).ToHexString();
+
+        internal static string GetSpanId(this ActivitySpanId activitySpanId)
+        {
+            var spanId = activitySpanId.ToHexString();
+            if (!string.Equals(spanId, INVALID_SPAN_ID, StringComparison.Ordinal))
+            {
+                return spanId;
+            }
+
+            return null;
+        }
+
+        internal static string GetTraceId(this ActivityTraceId activityTraceId)
+        {
+            var traceId = activityTraceId.ToHexString();
+            if (!string.Equals(traceId, INVALID_TRACE_ID, StringComparison.Ordinal))
+            {
+                return traceId;
+            }
+
+            return null;
+        }
+
+        internal static TelemetryType GetTelemetryType(this Activity activity)
+        {
+            if (activity.Kind == ActivityKind.Server || activity.Kind == ActivityKind.Consumer)
+            {
+                return TelemetryType.Request;
+            }
+            else
+            {
+                // TODO: If there a need, extend for other telemetry types.
+                return TelemetryType.Dependency;
+            }
+        }
+
+        // TODO: Change the return type to integer once .NET support it.
+        internal static string GetStatusCode(this Activity activity)
+        {
+            var status = activity.GetStatus().CanonicalCode switch
+            {
+                StatusCanonicalCode.Cancelled => 400,
+                StatusCanonicalCode.Unknown => 0,
```

Comment:
>StatusCanonicalCode.Unknown => 0, [](start = 16, length = 33)

Is this still needed as 0 is the default?

Reply:
we can completely ignore status for now - as OpenTelemetry is removing Status from spec.